Tasks/JenkinsDownloadArtifacts/task.loc.json

```diff
@@ -25,6 +25,12 @@
       "name": "advanced",
       "displayName": "ms-resource:loc.group.displayName.advanced",
       "isExpanded": false
+    },
+    {
+      "name": "azureStorage",
+      "displayName": "ms-resource:loc.group.displayName.azureStorage",
+      "isExpanded": true,
+      "visibleRule": "artifactProvider == Azure Storage"
```

Comment:
this should be "and propogatedArtifacts = true"

Reply:
i verified, the visibility toggle works fine without this.